### ISSUE TITLE
read colors from LS_COLORS

### ISSUE THREAD

**Issue by dansouza** [muesli/thunder#2]
perhaps an improvement would be to read (and use) the colors from LS_COLORS so "directories" have the right color?

**Comment by muesli** [comment]
Yeah, would be nice coloring the output! I think this requires some changes in upstream `ishell`, though.
